1.0/docs/devguide/properties.md

```diff
@@ -370,40 +370,59 @@ Example:
 In addition to properties, observers can also observe [paths to sub-properties](#observing-path-changes),
 [paths with wildcards](#deep-observation), or [array changes](#array-observation).
 
-### Observing path changes {#observing-path-changes}
+### Observing sub-property changes {#observing-path-changes}
 
-You can also observe changes to object sub-properties using the 
-`observers` array, by specifying a full path (`user.manager.name`)
-as a function argument.
+To observe changes in object sub-properties:
 
-Example:
-
-    Polymer({
+1.  Initialize your object to a default value.
```

Comment:
These aren't really steps (or have a necessary order), so they'd be better as bullet points than numbered items.

#1 here isn't actually necessary. (For example, imagine if your `user` object is only initialized when the user is logged in. You can still set up an observer for that path—it just won't fire until you set the object for the first time.)